tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunnerUtils.java

```diff
@@ -434,6 +441,9 @@ public static void validateError(String kind, Collection<Diagnostic> diagnostics
 
     public static void validateOutput(CompileResult compileResult, List<String> outputValues, List<Integer> lineNumbers,
                                       int absLineNum, ITestContext context) {
+        if (lineNumbers.isEmpty() || outputValues.isEmpty()) {
+            Assert.fail("Test output is empty");
+        }
```

Comment:
Doing this details of that test case will not be added to the report.

Reply:
Fixed this thanks